stream.js

```diff
@@ -155,7 +150,7 @@ function WebSocketStream(target, protocols, options) {
 
   function onmessage(event) {
     var data = event.data
-    if (data instanceof ArrayBuffer) data = Buffer.from(new Uint8Array(data))
```

Comment:
This change is unrelated and not needed but there is no need to instantiate a new `Uint8Array`. You can use the `ArrayBuffer` directly.

I can revert this change or move it to its own commit/PR if wanted.

Reply:
no need.